nautobot/core/templatetags/helpers.py

```diff
@@ -693,3 +693,13 @@ def hyperlinked_object_with_color(obj):
         content = f'<span class="label" style="color: {fgcolor(obj.color)}; background-color: #{obj.color}">{hyperlinked_object(obj)}</span>'
         return format_html(content)
     return "—"
+
+
+@register.filter()
+def get_obj_uuids(obj):
```

Comment:
Since this is new, I feel like the name is a tad of a misnomer seeing as it's expecting a queryset as input, not an object (which would imply an instance). Additionally, not all PKs will be UUIDs.

Perhaps `queryset_to_pks` or something?